code/src/containers/forms/product-form/ProductForm.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const productLocales = [
+  { key: "en", translationKey: "language.full.english" },
+  { key: "uk", translationKey: "language.full.ukrainian" }
+] as const;
+
+export const productCategories = [
+  { label: "productsAll.mobile", id: 1 },
+  { label: "productsAll.tablet", id: 2 },
+  { label: "productsAll.computer", id: 3 }
+];
```

Comment:
You can do similar thing here be restricting shape pf this object (by defining types for it)

Reply:
Thank you, fixed